ocr.py

```diff
@@ -27,13 +29,21 @@ def __init__(self, language='en', method=OCREngine.EASYOCR, detection_method=DET
         self.detection_method = detection_method
 
         if self.method == OCREngine.EASYOCR or self.detection_method == DETEngine.EASYOCR:
-            self.reader = easyocr.Reader(['en']) if language == 'en' else easyocr.Reader(['en', 'ja'])
+
+            if self.lang == 'en':
+
+                self.reader = easyocr.Reader(['en'], 
+                                             recog_network="en_v2")
+            if self.lang == 'jp':
+
+                self.reader = easyocr.Reader(['ja'],  
+                                             recog_network="jp_fine_tuned_ff4")
         
         self.openai_api = OpenAI_API()
         self.claude_api = Claude_API()
 
         if self.detection_method == DETEngine.FAST:
-            self.fast = TextDetectorFast('pretrained/fast_tiny_ic15_736_finetune_ic17mlt.pth',checkpoint='checkpoints/checkpoint_60ep.pth.tar')
+            self.fast = TextDetectorFast('pretrained/fast_tiny_ic15_736_finetune_ic17mlt.pth',checkpoint='checkpoints/checkpoint.pth.tar')
```

Comment:
why is the checkpoint in a tar format?

Reply:
I am not sure. I am reusing FAST inference code.